README.md

```diff
@@ -1 +1,87 @@
-# hello-microverse
\ No newline at end of file
+- 📗 Table of Contents
+- 📖 About the Project
+- 🛠 Built With
+- Tech Stack
+- Key Features
+- 🚀 Live Demo
+- 💻 Getting Started
+- Setup
+- Prerequisites
+- Install
+- Usage
+- Run tests
+- Deployment
+- 👥 Authors
+- 🔭 Future Features
+- 🤝 Contributing
+- ⭐️ Show your support
+- 🙏 Acknowledgements
+- ❓ FAQ
+- 📝 License
+
+
+📖 [Hello Microverse]
+This is to set up "Hellow Microverse Project"
+
+[Hello Microverse Project] is a set up project, to make sure we understand how to set up linters without errors, make sure that we use correct (GitHub flow) and make sure that we document our work properly.
+
+🛠 Built With
+Tech Stack
+- Major languages: html & css
+
+Key Features
+- linters Installation.
+- GitHub flow.
+- Proper documentation
+
+# Link
+https://github.com/Ellyboi/hello-microverse
+
+💻 Getting Started
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+In order to run this project you need:
+- Install github properly
+
+Setup
+- Clone this repository to your desired folder:
```

Comment:
[OPTIONAL] I think you can add this 'https://github.com/Ellyboi/hello-microverse' to this place, so that the users will know where to click when they want to clone your project 😊